demo/send/landingPages/9ed4f663-8687-4df2-8f35-a201231312b4.json

```diff
@@ -0,0 +1 @@
+{"Campaign":null,"SendToDetails":null,"Content":null,"OptIn":0,"SegmentsCount":0,"LastDayChanges":0,"ActiveSubscribers":0,"MailingList":null,"Entity":{"ConversionPercentage":0,"CreatedBy":"129.222.164.164","CreatedOn":"/Date(1643743574790-0000)/","Id":"9ed4f663-8687-4df2-8f35-a201231312b4","Name":"jf-test","PublishContext":"http://sitecoresendemo.sitecoresend.net/syPFQZ/jf-test","PublishContextNoProtocol":"sitecoresendemo.sitecoresend.net/syPFQZ/jf-test","PublishMode":1,"Status":0,"TotalConversions":0,"TotalPageViews":0,"TotalUniqueVisits":0,"Type":2,"sUpdatedBy":null,"UpdatedOn":"/Date(1658433456153-0000)/","UserId":"7dc7a9bd-0a14-427f-894a-66d8f7a938db","WebsiteId":"9ed4f663-8687-4df2-8f35-a201231312b4"}}
```

Comment:
is this "jf-test" really used by the solution and must be serialized here? Should it be renamed to something more meaningful if it is being used?